org.freecadweb.FreeCAD.yaml

```diff
@@ -1,11 +1,12 @@
 app-id: org.freecadweb.FreeCAD
-default-branch: stable
 base: io.qt.qtwebkit.BaseApp
 base-version: '5.14'
 runtime: org.kde.Platform
 runtime-version: '5.14'
 sdk: org.kde.Sdk
 command: FreeCAD
+desktop-file-name-prefix: "(beta) "
```

Comment:
I prefer `desktop-file-name-suffix` and use it for other betas already (shotcut, olive). Is there a Flathub consensus?

Reply:
1. if it is at the end I don't always see it in GNOME shell.
2. I have seen it elsewhere on flathub. Don't know about any consensus.